analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Constraints/NumberConstraintTest.cs

```diff
@@ -82,6 +82,16 @@ public void From_Object_OtherTypes()
             NumberConstraint.From((object)'*').Should().BeNull();
         }
 
+        [TestMethod]
+        public void IsSingleValue()
+        {
+            NumberConstraint.From(42).IsSingleValue.Should().BeTrue();
+            NumberConstraint.From(1, 1).IsSingleValue.Should().BeTrue();
+            NumberConstraint.From(null, null).IsSingleValue.Should().BeFalse();
+            NumberConstraint.From(null, 42).IsSingleValue.Should().BeFalse();
+            NumberConstraint.From(42, null).IsSingleValue.Should().BeFalse();
```

Comment:
I know coverage is not a requirement for this PR, but this makes sense to test.
```suggestion
            NumberConstraint.From(42, null).IsSingleValue.Should().BeFalse();
            NumberConstraint.Empty.IsSingleValue.Should().BeFalse();
```

Reply:
Oh! I think it's missing `From(1, 42)`.
Additional note: I just changed the behavior of `From(null, null)` in #7174. You should change the test to use `Empty` instead.